unicodetools/org/unicode/text/UCD/UCD_Names.java

```diff
@@ -1094,6 +1094,8 @@
         // Unicode 11, non-singletons:
         "Hanifi_Rohingya_Pa",
         "Hanifi_Rohingya_Kinna_Ya",
+        // Unicode 14
+        "THIN YEH"
```

Comment:
```suggestion
        "THIN_YEH"
```

or even `"Thin_Yeh"` which is the actual spelling of the value